api/applications/applicationRouter.js

```diff
@@ -3,6 +3,8 @@ const authRequired = require('../middleware/authRequired');
 const Application = require('./applicationModel');
 const Profile = require('../profile/profileModel');
 const router = express.Router();
+const { axios: axios } = require('axios');
```

Comment:
This line could be `const axios = require('axios').default;` - taken from the [axios docs](https://axios-http.com/docs/example).